src/pages/index.js

```diff
@@ -49,35 +65,115 @@ function handleImageClick(data) {
 }
 
 function handleEditSumbit(obj) {
+  popupEdit.renderLoading(true);
   userInfo.setUserInfo(obj);
-  popupEdit.close();
-  formEditValidator.disableButton();
+  api
+    .updateCurrentUser(
+      JSON.stringify({
+        name: obj.name,
+        about: obj.about,
+      })
+    )
+    .then(() => {
+      popupEdit.close();
+      popupEdit.renderLoading(false);
```

Comment:
`      popupEdit.renderLoading(false);
`
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.


**Please, correct it everywhere in the code**